src/pkg/repository/repository.go

```diff
@@ -329,7 +330,7 @@ func (r repository) NewBackupWithLookup(
 		gc,
 		r.Account,
 		sel,
-		sel,
+		inMock.NewProvider(ownerID, ownerName),
```

Comment:
this shouldn't be using the mock since it's prod code

Reply:
oh, wow, how did I miss that?